plugins/Tour/Engagement/ChallengeDisableBrowserArchiving.php

```diff
@@ -0,0 +1,37 @@
+<?php
+/**
+ * Matomo - free/libre analytics platform
+ *
+ * @link https://matomo.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ *
+ */
+namespace Piwik\Plugins\Tour\Engagement;
+
+use Piwik\ArchiveProcessor\Rules;
+use Piwik\Piwik;
+
+class ChallengeDisableBrowserArchiving extends Challenge
+{
+    public function getName()
+    {
+        return Piwik::translate('Tour_DisableBrowserArchiving');
+    }
+
+    public function getId()
+    {
+        return 'disable_browser_archiving';
+    }
+
+    public function isCompleted()
+    {
+        return !Rules::isBrowserTriggerEnabled();
+    }
+
+    public function getUrl()
+    {
+        return 'https://matomo.org/docs/setup-auto-archiving/';
+    }
```

Comment:
Would it be better to link to the settings page here and put the matomo docs URL in the settings page/

Reply:
Edit: same question applies to some of the other tasks like 'view visits log'

Reply:
Was thinking about this as well but then the guides explain it a lot better and gives them more insights than the settings itself.